packages/document/main-doc/docs/en/blog/index.md

```diff
@@ -4,5 +4,70 @@ sidebar_position: 1
 
 # Blog
 
-You can find the latest news about Modern.js here.
+Find the latest update of modern.js and our thought processes here. Occasionally we explain the infrastructure behind it.
 
+---
+
+## Releasing MOdern.js v2: support Rspack
+
+> Published on 16.03.2023
+
+Hiyaaaaa~~ we are excited to announce the release of Modern.js v2!
+
+Modern.js is a suite of infrastructure tools we built for web development inside Bytedance (we call ourself Web-Infra-Dev internally...). Since we open sourced this project a little more than a year ago, there were dozens of contributors helped us on development; we aggregated more than 2,000 pull requests and modern.js started to support build tool like Rspack, nested routes, SSR(Streaming Server Render).
+
+We are extremely proud of what we have achieved so far, you can also refer to [this article](/blog/updates/v2-release-note)👈🏻 to see what has changed over the last year in modern.js.
+
+---
+
+## What is SSR(Streaming Server Rendering) in React 18
+
+> Published on 16.12.2022
+
+Since React 18, react supports a new type of SSR(streaming server rendering) and it brought two advantages over the react tool chain:
+
+- Streaming HTML: Server will be able to transmit HTML to browser parts by parts, rather than waiting until the whole page being rendered. Client side will render the page faster thus dramatically increase performance benchmark scores like TTFB(Time to First Byte), FCP(First Contentful Paint) and more.
+
+- Selective Hydration: On the client side, browser can hydrate only the HTML elements that has already been rendered, without needing to wait until the whole page finish rendering and all the javascript bundle being loaded.
+
+To understand its design further, check out this [Github Discussion](https://github.com/reactwg/react-18/discussions/37) by Dan Abramov or watch this [talk](https://www.youtube.com/watch?v=pj5N-Khihgc).
+
+Or read more from us at [here](react-streaming-ssr).
```

Comment:
it looks like a broken link

Reply:
```suggestion
Or read more from us at [here](/blog/react-streaming-ssr).
```

Reply:
This is another translated article I thought I might need to work on but haven't done. Its origin was the article published on WeChat BytedanceFE

Reply:
It would be great if we could translate the article from WeChat BytedanceFE